packages/block-library/src/image/style.scss

```diff
@@ -186,23 +186,36 @@
 	visibility: hidden;
 
 	.close-button {
-		font-size: 40px;
 		position: absolute;
-		top: 20px;
-		right: 20px;
+		top: 12.5px;
+		right: 12.5px;
+		padding: 0;
 		cursor: pointer;
 		z-index: 5000000;
 	}
 
 	.wp-block-image {
-		display: flex;
-		justify-content: center;
-		align-items: center;
 		width: 100%;
 		height: 100%;
-		z-index: 3000000;
 		position: absolute;
+		z-index: 3000000;
+		box-sizing: border-box;
+		display: flex;
+		justify-content: center;
+		align-items: center;
 		flex-direction: column;
+		padding: 30px;
+
+		figcaption {
+			margin: 10px 0 0 0;
```

Comment:
I guess that, if we are using `display: none`, we don't need this margin, right?